2-mandatory/1-writers.js

```diff
@@ -71,7 +77,15 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let arr = writers;
+  let result = arr.map((e) => {
+    e.age >= 40 && e.age <= 49 && e.alive === false
+      ? console.log(
+          `Writer ${e.firstName} ${e.lastName} died at ${e.age} years old.`
+        )
+      : null;
+  });
+  return result;
```

Comment:
This works well. You don't need to reassign "writers" to "arr". you can call map directly on "writers", unless you see some advantage to doing it your way.